src/sentry/snuba/metrics/fields/base.py

```diff
@@ -291,6 +292,10 @@ class MetricOperation(MetricOperationDefinition, ABC):
     def validate_can_orderby(self) -> None:
         raise NotImplementedError
 
+    @abstractmethod
+    def validate_can_groupby(self) -> bool:
```

Comment:
if it's just True/False we can try to use https://docs.python.org/3/library/abc.html#abc.abstractproperty so later you can write
if self.validate_can_groupby:
    pass
but since other valicate_can* are abstract methods are good.

Reply:
oo TIL nice but yeah it follows the pattern of other validation methods

Reply:
that's why i said we are good to go without making any changes/